test/smoke/testApps/SimpleEventData/src/main/java/com/microsoft/ajl/simplecalc/ICalculationHistoryService.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.ajl.simplecalc;
+
+import java.util.Collection;
+
+import com.microsoft.ajl.simplecalc.model.BinaryCalculation;
+import com.microsoft.ajl.simplecalc.model.BinaryCalculation.TimestampedBinaryCalculation;
+
+public interface ICalculationHistoryService {
```

Comment:
I don't think either of those things are necessary.

The java standard for interfaces is to suffix with -able. I'm not a fan. This is fine. (not to mention this file is not doing anything at the moment).

There's no use case in the context of this interface where it would be necessary to use List.